### ISSUE TITLE
Tabs deep links

### ISSUE THREAD

**Issue by decabeza** [consul/consul#2210]
Background
===

When a user visit a proposal or budget investment and clic on a comments, notifications or documents link the tabs works correctly but when  a user visit for example `proposals/55-ut-impedit-consequatur#tab-documents` the active tab it's still the first one (comments in this case)

What we need
===

- Use `data-deep-link="true"`, on these tabs [see Foundation docs]( https://foundation.zurb.com/sites/docs/tabs.html).

- When a user clic on a tab should add `#tab-x` to url.

- When a user navigate to `proposals/55-ut-impedit-consequatur#tab-X` should show the correct tab as active.

Notes
===

- This issue seems be generated by **turbolinks**
- One [similar issue](https://github.com/zurb/foundation-rails/issues/248), maybe for reference

**Comment by iagirre** [comment]
I'll do this

**Comment by iagirre** [comment]
After been hitting myself with a wall once and again trying to figure out how to resolve the bug, I realized that the deep linking feature doesn't exists in the foundation-rails v6.2.4.0 (the one that the app is using).
I've compared the file `foundation.tabs.js` from this version with the one in v6.3.0.0. Version 6.3 already has it implemented.
I've tried updating the gem (commenting the code that fails, just to try the functionality) and it works from v6.3.1.0 on, even with turbolinks.
Now, I'm looking for some kind of patch to fix this issue until the gem can be updated (#1673)
